packages/core/src/apiProviders/utils.ts

```diff
@@ -71,8 +68,10 @@ const manageError = (
         type: 'error',
         position: 'bottom',
         bottomOffset: 20,
-        text1: `Error ${code}`,
-        text2: `Failed to ${action}: ${message}.`,
+        text1: i18nProvider.i18n.t('Base_Error'),
+        text2: ` ${i18nProvider.i18n.t(
+          'Base_Failed_To',
+        )} ${action}: ${message}.`,
```

Comment:
As action is something given in function props, I think we can do better : 
- ask in props for a translation key
- display translated key

Caution: this implies changing all use of this function by transforming string to translation key